mPower2/mPower2/Profile/ProfileTableViewCell.swift

```diff
@@ -36,9 +36,10 @@ import Research
 
 class ProfileTableViewCell: UITableViewCell {
     
+    @IBOutlet weak var icon: UIImageView?
```

Comment:
You added an outlet, but I don't see it being used?

Reply:
Aah thank you! I was going to look into why that wasn't working this morning.